examples/oneapi/cpp/source/decision_forest/df_reg_dense_batch.cpp

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "example_util/utils.hpp"
+#include "oneapi/dal/algo/decision_forest.hpp"
+
+using namespace oneapi;
+namespace df = oneapi::dal::decision_forest;
+
+int main(int argc, char const *argv[]) {
+  constexpr std::int64_t tree_count = 10;
+  constexpr std::int64_t features_per_node = 1;
+  constexpr std::int64_t min_observations_in_leaf_node = 2;
```

Comment:
It makes no sense to put them in separate names.  Use constants directly, as in other algorithms.

Reply:
removed